Fix:
error: implicit declaration of function 'mkdtemp' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
error: implicit declaration of function 'mkstemps' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
https://github.com/derf/feh/pull/647
--- src/imlib.c.orig	2022-01-03 04:34:57.000000000 -0600
+++ src/imlib.c	2022-01-08 15:49:42.000000000 -0600
@@ -35,6 +35,7 @@
 #include <netinet/in.h>
 #include <arpa/inet.h>
 #include <netdb.h>
+#include <unistd.h>
 
 #ifdef HAVE_LIBCURL
 #include <curl/curl.h>
